### PR TITLE
Fix #315, to update 10.2.8 and its subsections

### LINKED ISSUE #315: Need to update the section 10.2.8 and its subsections

### PR DESCRIPTION
**Pull request by sunghee-hwang**
***
<a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/pull/318.html" title="Last updated on Mar 20, 2023, 10:48 PM UTC (f465ca6)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/AOMediaCodec/iamf/318/4621c48...f465ca6.html" title="Last updated on Mar 20, 2023, 10:48 PM UTC (f465ca6)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -2786,18 +2786,15 @@ Step3: Generate IA sequence which starts descriptor OBUs and followed by Tempora
...
-//To Do: Fill in how to generate loudness_config()
-
-#### DRC Config ####  {#iamfgeneration-postprocessing-drc}
-
-This section provide a guideline to generate drc_config().
-
-//To Do: Fill in how to generate drc_config()
+#### Loudness Information ####  {#iamfgeneration-postprocessing-loudness}
 
+This section provide a guideline to generate loudness_info().
```
Comment (yeroro) [inline index.bs:2791]: "provides"

File: index.bs [hunk 2]
```diff
@@ -2786,18 +2786,15 @@ Step3: Generate IA sequence which starts descriptor OBUs and followed by Tempora
...
+		
+For a given Mix Presentation OBU and a given [=loudness_layout=] of a sub-mix of the given Mix Presentation OBU, followings are processed in order to produce loudness_info().
```
Comment (yeroro) [inline index.bs:2793]: "the" followings

File: index.bs [hunk 3]
```diff
@@ -2786,18 +2786,15 @@ Step3: Generate IA sequence which starts descriptor OBUs and followed by Tempora
...
+- Each of Audio Elements specified in the sub-mix of the give Mix Presentation OBU is rendered to the given [=loudness_layout=] according to [=rendering_config()=] for the Audio Element.
```
Comment (yeroro) [inline index.bs:2794]: "Each Audio Element" or "Each of the Audio Elements"

File: index.bs [hunk 4]
```diff
@@ -2786,18 +2786,15 @@ Step3: Generate IA sequence which starts descriptor OBUs and followed by Tempora
...
+- Each of Audio Elements specified in the sub-mix of the give Mix Presentation OBU applies [=mix_gain=] according to [=element_mix_config()=] for the Audio Element.
+- All of Audio Elements specified in the sub-mix of the give Mix Presentation OBU are summed, and then applies [=mix_gain=] according to [=output_mix_config()=] for the sub-mix.
```
Comment (yeroro) [inline index.bs:2796]: "All Audio Elements" or "All of the Audio Elements"

File: index.bs [hunk 5]
```diff
@@ -2786,18 +2786,15 @@ Step3: Generate IA sequence which starts descriptor OBUs and followed by Tempora
...
-This section provide a guideline to generate drc_config().
-
-//To Do: Fill in how to generate drc_config()
+#### Loudness Information ####  {#iamfgeneration-postprocessing-loudness}
 
+This section provide a guideline to generate loudness_info().
+		
+For a given Mix Presentation OBU and a given [=loudness_layout=] of a sub-mix of the given Mix Presentation OBU, followings are processed in order to produce loudness_info().
+- Each of Audio Elements specified in the sub-mix of the give Mix Presentation OBU is rendered to the given [=loudness_layout=] according to [=rendering_config()=] for the Audio Element.
+- Each of Audio Elements specified in the sub-mix of the give Mix Presentation OBU applies [=mix_gain=] according to [=element_mix_config()=] for the Audio Element.
```
Comment (yeroro) [inline index.bs:2795]: s/give/given

**Review by yeroro — CHANGES_REQUESTED** [review]

**Comment by tdaede** [comment]
Mergeable with yeroro's editorial comments addressed.

File: index.bs [hunk 6]
```diff
@@ -2788,12 +2788,12 @@ This section provides a guideline to generate algorithms for post processing.
...
+This section provides a guideline to generate loudness_info().
 		
-For a given Mix Presentation OBU and a given [=loudness_layout=] of a sub-mix of the given Mix Presentation OBU, followings are processed in order to produce loudness_info().
-- Each of Audio Elements specified in the sub-mix of the give Mix Presentation OBU is rendered to the given [=loudness_layout=] according to [=rendering_config()=] for the Audio Element.
-- Each of Audio Elements specified in the sub-mix of the give Mix Presentation OBU applies [=mix_gain=] according to [=element_mix_config()=] for the Audio Element.
-- All of Audio Elements specified in the sub-mix of the give Mix Presentation OBU are summed, and then applies [=mix_gain=] according to [=output_mix_config()=] for the sub-mix.
+For a given Mix Presentation OBU and a given [=loudness_layout=] of a sub-mix of the given Mix Presentation OBU, the followings are processed in order to produce loudness_info().
+- Each of Audio Elements specified in the sub-mix of the given Mix Presentation OBU is rendered to the given [=loudness_layout=] according to [=rendering_config()=] for the Audio Element.
```
Comment (yeroro) [inline index.bs:2794]: "Each Audio Element" or "Each of the Audio Elements"

File: index.bs [hunk 7]
```diff
@@ -2788,12 +2788,12 @@ This section provides a guideline to generate algorithms for post processing.
...
+- Each of Audio Elements specified in the sub-mix of the given Mix Presentation OBU applies [=mix_gain=] according to [=element_mix_config()=] for the Audio Element.
+- All of Audio Elements specified in the sub-mix of the given Mix Presentation OBU are summed, and then applies [=mix_gain=] according to [=output_mix_config()=] for the sub-mix.
```
Comment (yeroro) [inline index.bs:2796]: "All Audio Elements" or "All of the Audio Elements"

**Review by yeroro — APPROVED** [review]
Approve after the editorial changes.

**Comment by tdaede** [comment]
This one still has a couple editorial comments.

**Pull request merged**